string folder
---
# Success if empty
string error
